test/common.js

```diff
@@ -654,3 +654,29 @@ exports.skipIfInspectorDisabled = function skipIfInspectorDisabled() {
     process.exit(0);
   }
 };
+
+const arrayBufferViews = [
+  Int8Array,
+  Uint8Array,
+  Uint8ClampedArray,
+  Int16Array,
+  Uint16Array,
+  Int32Array,
+  Uint32Array,
+  Float32Array,
+  Float64Array,
+  DataView
+];
+
+exports.getArrayBufferViews = function getArrayBufferViews(buf) {
+  const { buffer, byteOffset, byteLength } = buf;
+
+  const out = [];
+  for (const type of arrayBufferViews) {
+    const { BYTES_PER_ELEMENT = 1 } = type;
+    if (Number.isInteger(byteLength % BYTES_PER_ELEMENT)) {
```

Comment:
Did you mean `byteLength % BYTES_PER_ELEMENT === 0`, or `Number.isInteger(byteLength / BYTES_PER_ELEMENT)`?